### PR TITLE
Feature/sig 924 splitsen van meldingen with models

### PR DESCRIPTION
**Pull request by jasperswart**

### CODE REVIEW

File: src/signals/incident-management/containers/IncidentSplitContainer/components/SplitForm/index.test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// import React from 'react';
+// import { shallow } from 'enzyme';
+
+// import SplitForm from './index';
+
+describe('<SplitForm />', () => {
+  it('Expect to have unit tests specified', () => {
+    expect(true).toEqual(true);
```
Comment (RobertLodder) [inline src/signals/incident-management/containers/IncidentSplitContainer/components/SplitForm/index.test.js]: Testing nothing

File: src/signals/incident-management/containers/IncidentSplitContainer/saga.test.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  // it('should fetchIncident error', () => {
+  //   const id = 1000;
+  //   const action = { payload: id };
+  //   const error = new Error('404 Not Found');
+
+  //   const gen = fetchIncident(action);
+  //   gen.next();
+  //   expect(gen.throw(error).value).toEqual(put(requestIncidentError(error))); // eslint-disable-line redux-saga/yield-effects
+  // });
```
Comment (RobertLodder) [inline src/signals/incident-management/containers/IncidentSplitContainer/saga.test.js]: Commented out code

**Comment by RobertLodder** [comment]
In the future try to create a separate pull request for separate features like the generator update or the model refactoring. That prevents this from becoming a huge PR.

File: src/shared/services/api/api.test.js [hunk 3]
```diff
@@ -94,34 +94,65 @@ describe('api  service', () => {
...
+  //   //     headers: {
+  //   //       'Content-Type': 'application/json'
+  //   //     },
+  //   //     body: JSON.stringify(params)
+  //   //   };
+  //   //   const gen = authPostCall(url, params);
+  //   //   expect(gen.next().value).toEqual(select(makeSelectAccessToken())); // eslint-disable-line redux-saga/yield-effects
+  //   //   expect(gen.next().value).toEqual(call(request, fullUrl, options)); // eslint-disable-line redux-saga/yield-effects
+  //   // });
+  // });
```
[the remark this answers is not in the document]
Comment (jasperswart) [reply src/shared/services/api/api.test.js]: gedaan

File: src/signals/incident-management/containers/IncidentDetailPage/components/SplitNotificationBar/index.test.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+// import React from 'react';
+// import { shallow } from 'enzyme';
+
+// import NotificationBar from 'index';
+
+describe('<NotificationBar />', () => {
+  it('Expect to have unit tests specified', () => {
+    expect(true).toEqual(true);
```
[the remark this answers is not in the document]
Comment (jasperswart) [reply src/signals/incident-management/containers/IncidentDetailPage/components/SplitNotificationBar/index.test.js]: gedaan

File: src/signals/incident-management/containers/IncidentDetailPage/index.js [hunk 5]
```diff
@@ -56,14 +57,19 @@ export class IncidentDetailPage extends React.Component { // eslint-disable-line
     this.setState({ printView: !this.state.printView });
   }
 
+  onResetSplit() {
```
[the remark this answers is not in the document]
Comment (jasperswart) [reply src/signals/incident-management/containers/IncidentDetailPage/index.js]: now: onResetSplitState

File: src/signals/incident-management/components/CopyFileInput/index.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './style.scss';
+
+export const TextInput = (props) => {
```
[the remark this answers is not in the document]
Comment (jasperswart) [reply src/signals/incident-management/components/CopyFileInput/index.js]: gedaan

File: src/signals/incident-management/containers/IncidentSplitContainer/components/IncidentPart/inxed.test.js [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+  //     const wrapper = shallow(
+  //       <IncidentPart {...props} />
+  //     );
+  //     expect(wrapper).toMatchSnapshot();
+  //     // expect(props.onRequestIncident).toHaveBeenCalledWith('42');
+  //   });
+  // });
+
+  it('shoild', () => {
+    expect(true).toBe(true);
```
[the remark this answers is not in the document]
Comment (jasperswart) [reply src/signals/incident-management/containers/IncidentSplitContainer/components/IncidentPart/inxed.test.js]: gedaan

File: src/signals/incident-management/containers/IncidentSplitContainer/components/SplitForm/index.test.js (hunk 1 above)
Comment (jasperswart) [reply src/signals/incident-management/containers/IncidentSplitContainer/components/SplitForm/index.test.js]: gedaan

File: src/signals/incident-management/containers/IncidentSplitContainer/saga.test.js (hunk 2 above)
Comment (jasperswart) [reply src/signals/incident-management/containers/IncidentSplitContainer/saga.test.js]: gedaan

**Pull request merged**